File: java/src/Ice/Dispatcher.java
// **********************************************************************
//
// Copyright (c) 2001
// MutableRealms, Inc.
// Huntsville, AL, USA
//
// All Rights Reserved
//
// **********************************************************************

package Ice;

public interface Dispatcher
{
    IceInternal.DispatchStatus
    __dispatch(IceInternal.Incoming in, Current current);
}

File: cs/src/Ice/Dispatcher.cs
// **********************************************************************
//
// Copyright (c) 2003-2009 ZeroC, Inc. All rights reserved.
//
// This copy of Ice is licensed to you under the terms described in the
// ICE_LICENSE file included in this distribution.
//
// **********************************************************************

namespace Ice
{
    public delegate void DispatcherCall();
    
    public interface Dispatcher
    {
        void dispatch(DispatcherCall call, Ice.Connection con);
    }
}